ci/jobs/pulp-dev.yaml

```diff
@@ -69,13 +73,17 @@
             build-number: ${{TRIGGERED_BUILD_NUMBER_PULP_SMASH_RUNNER}}
             flatten: true
         - shell: |
-            if [ "{pulp_version}" = "2.8" ]; then
+            if [ "${{PULP_VERSION}}" = "2.8" ]; then
```

Comment:
Converted this a environment variable to avoid seen on jobs something like `if [ "2.8" = "2.8" ]` hard coded.